### PR TITLE
track view mode

### PR DESCRIPTION
**Pull request by grimm**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -56,14 +56,15 @@
...
-    t.datetime "updated_at",                            null: false
+    t.datetime "created_at",                                            null: false
+    t.datetime "updated_at",                                            null: false
     t.decimal  "center_lat",   precision: 12, scale: 8
     t.decimal  "center_lng",   precision: 12, scale: 8
     t.decimal  "zoom",         precision: 12, scale: 8
     t.string   "presenter_id"
     t.string   "basemap"
     t.integer  "workspace_id"
+    t.boolean  "view_mode",                             default: false
```
Comment (teknofire) [inline db/schema.rb]: Lets rename this field to be `3d_view_mode`

Comment (grimm) [inline db/schema.rb]: Looks like you can't start a postgres column with a number.  view_mode_3d does work though.

Comment (grimm) [inline db/schema.rb]: Going with view_3d_mode, sounds better.

File: app/views/views/_form.html.haml [hunk 2]
```diff
@@ -13,6 +13,10 @@
         = f.input :center_lat
       .col-sm-4
         = f.input :zoom
+    .row
+      .col-sm-12
+        = f.label 'Use 3D mode'
+        = f.check_box :view_3d_mode, lable: 'Use 3D mode'
```
Comment (teknofire) [inline app/views/views/_form.html.haml]: Should be using the simple form input method, if you need to override the label there is a param you can specify.

**Pull request merged**